### PR TITLE
Issue #89 "Option 2" Solution Implemented

### PR DESCRIPTION
**Pull request by NatMills**
See: https://github.com/linkagescape/linkage-mapper/issues/89

### CODE REVIEW

**Comment by dkav** [comment]
I would suggest [squashing](https://support.gitkraken.com/working-with-commits/squash/) the changes into one commit. It reduces the noise and size (due to the tbx binary).

File: toolbox/scripts/lp_main.py [hunk 1]
```diff
@@ -313,6 +313,12 @@ def clim_priority_values(lcp_lines):
     canalog_r_min, canalog_r_max = value_range(lcp_lines, "CAnalog_Ratio")
     cprefer_r_min, cprefer_r_max = value_range(lcp_lines, "CPrefer_Ratio")
 
+    if lm_env.MINRMAX > canalog_r_max:
+        canalog_r_max = lm_env.MINRMAX
+    else:
```
Comment (dkav) [inline toolbox/scripts/lp_main.py]: `else:` is redundant.

**Comment by johngallo** [comment]
Good idea about squashing., and thanks for the link to the video!  I'll give Nat a chance to do that.  (He is traveling today I believe, and may be for several days). @NatMills you can do that based on the video Darren shared, and hopefully it allows you to squash both your commits and mine all into one.  In fact I see one more change I can do at this point regarding the multi-tool combo.

**Comment by dkav** [comment]
Squashed and updated changes forthcoming in new pull request.

**Comment by johngallo** [comment]
OK, thanks @dkav !  When you squash the commits into one commit, does it concatenate the names of all the commits into one, and the descriptions of all the commits into one, or does it just take one of the names and one of the descriptions?  If the latter, you let me know and I can paste the other descirptions into the commit description once you push it to github, while I still have them.  Also, the commits were by two people, Nat and myself.  Who does it say the commit is by?  If it says me, can we add into the description that it was mostly Nat that did the work so his credit gets documented?

**Comment by dkav** [comment]
See commit 43cc3f2304e51fb6e01c1762aff2be4689b7e928. Let me know if this works.

**Comment by johngallo** [comment]
Yes, this works. In the future it will be ideal if each person squashes their own commits.  (But we have to be careful to not have too much material in one commit).  On another note, is there something different about the new climate_curves branch? It is not showing up for me in GitKraken like new branches normally do, but maybe I am rusty on something there.

**Pull request closed without merging**